src/watchdog/observers/inotify.py

```diff
@@ -170,6 +171,13 @@ def queue_events(self, timeout, full_events=False):
                 cls = DirCreatedEvent if event.is_directory else FileCreatedEvent
                 self.queue_event(cls(src_path))
                 self.queue_event(DirModifiedEvent(os.path.dirname(src_path)))
+            elif event.is_close_write and not event.is_directory:
+                cls = FileClosedEvent
+                self.queue_event(cls(src_path))
+                self.queue_event(DirModifiedEvent(os.path.dirname(src_path)))
+            elif event.is_close_nowrite and not event.is_directory:
+                cls = FileClosedEvent
+                self.queue_event(cls(src_path))
```

Comment:
```suggestion
            # elif event.is_close_nowrite and not event.is_directory:
            #     cls = FileClosedEvent
            #     self.queue_event(cls(src_path))
```

Reply:
Yeah ok I will rebase 66c1e00 to have atomic changes about the removing of this event